crates/astria-sequencer/src/mempool/transactions_container.rs

```diff
@@ -159,6 +174,86 @@ impl PendingTransactionsForAccount {
     fn highest_nonce(&self) -> Option<u32> {
         self.txs.last_key_value().map(|(nonce, _)| *nonce)
     }
+
+    /// Removes and returns transactions that exceed the balances in `available_balances`.
+    fn find_demotables(
+        &mut self,
+        available_balances: &HashMap<IbcPrefixed, u128>,
```

Comment:
```suggestion
        available_balances: HashMap<IbcPrefixed, u128>,
```

think you can just pass the map here and not clone below